test/OrchardCore.Tests/Apis/Context/BlogPostDeploymentContext.cs

```diff
@@ -65,37 +65,32 @@ public JObject GetContentStepRecipe(ContentItem contentItem, Action<JObject> mut
 
         public async Task<HttpResponseMessage> PostRecipeAsync(JObject recipe, bool ensureSuccess = true)
         {
-            using (var zipStream = new MemoryStream())
+            using var zipStream = new MemoryStream();
+            using (var zip = new ZipArchive(zipStream, ZipArchiveMode.Create, true))
             {
-                using (var zip = new ZipArchive(zipStream, ZipArchiveMode.Create, true))
-                {
-                    var entry = zip.CreateEntry("Recipe.json");
-                    using (var streamWriter = new StreamWriter(entry.Open()))
-                    {
-                        using (var jsonWriter = new JsonTextWriter(streamWriter))
-                        {
-                            await recipe.WriteToAsync(jsonWriter);
-                            await jsonWriter.FlushAsync();
-                        }
-                    }
-                }
-                zipStream.Position = 0;
+                var entry = zip.CreateEntry("Recipe.json");
+                using var streamWriter = new StreamWriter(entry.Open());
+                using var jsonWriter = new JsonTextWriter(streamWriter);
+                await recipe.WriteToAsync(jsonWriter);
+                await jsonWriter.FlushAsync();
+            }
 
-                using (var requestContent = new MultipartFormDataContent())
-                {
-                    requestContent.Add(new StreamContent(zipStream), nameof(ImportViewModel.Content), "Recipe.zip");
-                    requestContent.Add(new StringContent(RemoteDeploymentClientName), nameof(ImportViewModel.ClientName));
-                    requestContent.Add(new StringContent(RemoteDeploymentApiKey), nameof(ImportViewModel.ApiKey));
+            zipStream.Position = 0;
```

Comment:
Microsoft seems to be using Seek method instead is setting the position to 0. Not sure if there is advantage or not

Reply:
Yes, we could use `zipStream.Seek(0, SeekOrigin.Begin)` which does exactly the same but with an additional check on the sum of the `_origin` and the passed `offset` (here = 0), so to just clear the position that's okay to use the setter that also does the same other checks.